Xamarin.Forms.Platform.WPF/Renderers/ListViewRenderer.cs

```diff
@@ -125,6 +135,31 @@ void UpdateItemSource()
 			}
 		}
 
+		WpfScrollBarVisibility ScrollBarVisibilityToWpf(ScrollBarVisibility visibility)
+		{
+			switch (visibility)
+			{
+				case ScrollBarVisibility.Always:
+					return WpfScrollBarVisibility.Visible;
+				case ScrollBarVisibility.Default:
+					return WpfScrollBarVisibility.Auto;
+				case ScrollBarVisibility.Never:
+					return WpfScrollBarVisibility.Hidden;
+				default:
+					return WpfScrollBarVisibility.Auto;
+			}
+		}
+
+		void UpdateVerticalScrollBarVisibility()
```

Comment:
Shouldn't these use the `ToWpfScrollBarVisibility` extension method? And then you could remove `ScrollBarVisibilityToWpf` entirely.